src/lambda/utils.ts

```diff
@@ -44,11 +44,16 @@ export async function* listLambdaFunctions(client: LambdaClient): AsyncIterableI
 }
 
 /**
- * Converts Lambda handler into a filename by stripping the function name and appending the correct file extension.
+ * Parses Lambda handler into a filename by stripping the function name and appending the correct file extension and a function name
  * Only works for supported languages (Python/JS)
  * @param configuration Lambda configuration object from getFunction
  */
-export function getLambdaFileNameFromHandler(configuration: Lambda.FunctionConfiguration): string {
+export function getLambdaDetailsFromConfiguration(
```

Comment:
name can be shorter without a loss of clarity (and the type system says what parameter it needs already)

```suggestion
export function getLambdaDetails(
```